### PR TITLE
Small fixes to viewer and engine example

### PR DESCRIPTION
**Pull request by slimbuck**
Changes in this PR:
- squash lint warnings in the engine viewer
- get text-canvas-font example working with the visual tester

### CODE REVIEW

File: examples/user-interface/text-canvas-font.html [hunk 1]
```diff
@@ -58,7 +58,7 @@
         app.assets.add(fontAsset);
         app.assets.load(fontAsset);
         app.assets.on('load:' + fontAsset.id, function () {
-            app.start();
+            
         });
```
Comment (willeastcott) [inline examples/user-interface/text-canvas-font.html]: You might as well delete this event handler if it doesn't do anything.

File: tools/glb-viewer/src/viewer.js [hunk 2]
```diff
@@ -308,8 +309,10 @@ Object.assign(Viewer.prototype, {
     }
 });
 
-var viewer;
+/* eslint-disable no-unused-vars */
 
 var main = function () {
-    viewer = new Viewer(document.getElementById("application-canvas"));
+    new Viewer(document.getElementById("application-canvas"));
```
Comment (willeastcott) [inline tools/glb-viewer/src/viewer.js]: I'm not a big fan of not assigning the result of calling a constructor to anything.

Comment (slimbuck) [reply tools/glb-viewer/src/viewer.js]: Try telling our lint overlord so! Would you prefer I disable that check here instead?

Comment (slimbuck) [reply tools/glb-viewer/src/viewer.js]: Oh I see it's already in that block (due to main being called from index.html).

File: examples/user-interface/text-canvas-font.html (hunk 1 above)
Comment (slimbuck) [reply examples/user-interface/text-canvas-font.html]: yep, done

File: tools/glb-viewer/src/viewer.js (hunk 2 above)
Comment (slimbuck) [reply tools/glb-viewer/src/viewer.js]: it's back!

**Pull request merged**